### PR TITLE
Accessibility update

### PR DESCRIPTION
**Pull request by Jord4N-AC**
Change submit background color based on WebDev Tool-Bar results

### CODE REVIEW

**Comment by zkr024** [comment]
# title:
	-Portfolio: passed.

# Image text alternatives:
	-10 alerts: alt message repeated in media icons and the images from the abilities class.

	Actions:
		HTML file lines 38-42 and lines 197-201.
		Had to modify the alt contect to clarify the information about each icon.

# Text headings:
	-36 Structural elements - passed

# Color contrast:
	-12 Contrast errors: 

	Access button (background color - Font color)
	Actions:
		CSS file line 363. Modify the background color from #ff6b00 to #823803 in the class access_button and submit.
		CSS file line 364. Add outline with color #ff6b00 to keep the site style.
		
	Cards background and Font color (desktop version)
	Actions:
		CSS file line 540 and line 576. Modify the background to make it darker. That would make the font visible.

# Resize:
	-The site can adapt when changing its size allowing to read and check the information.

# Interaction
	- tab - woks
	- shif + tab - works
	- arrows - scrol up/down - works

# Moving content and Multimedia
	-NA

# Basic Structure
	-Header - body - footer.

File: index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@ <h2 class="title">About me</h2>
                 <ul class="details">
                     <li><ul class="abilities">
                         <li class="abilities_title one">
-                            <img src="images/rectangle.svg" alt="Icon taken from figma">
+                            <img src="images/rectangle.svg" alt="Rhombus icon taken from figma">
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly this kind of image it's like icons so it's not important to set the `alt` attribute here. it is just decoration or "eye candy" — so you can set it to null `(alt="")`.

File: index.html [hunk 2]
```diff
@@ -146,7 +146,7 @@ <h3>Language</h3>
                     </ul></li>
                     <li><ul class="abilities">
                         <li class="abilities_title two">
-                            <img src="images/square.svg" alt="Icon taken from figma">
+                            <img src="images/square.svg" alt="Square icon taken from figma">
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly this kind of image it's like icons so it's not important to set the `alt` attribute here. it is just decoration or "eye candy" — so you can set it to null `(alt="")`.

File: index.html [hunk 3]
```diff
@@ -157,7 +157,7 @@ <h3>Frameworks</h3>
                     </ul></li>
                     <li><ul class="abilities">
                         <li class="abilities_title three">
-                            <img src="images/circle.svg" alt="Icon taken from figma">
+                            <img src="images/circle.svg" alt="Circle icon taken from figma">
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly this kind of image it's like icons so it's not important to set the `alt` attribute here. it is just decoration or "eye candy" — so you can set it to null `(alt="")`.

File: index.html [hunk 4]
```diff
@@ -171,27 +171,34 @@ <h3>Skills</h3>
...
-                    <li><a href="#"><img src="images/darkgithub.svg" alt="github icon taken from figma"></a></li>
-                    <li><a href="#"><img src="images/darkin.svg" alt="icon taken from figma"></a></li>
-                    <li><a href="#"><img src="images/darkok.svg" alt="icon taken from figma"></a></li>
-                    <li><a href="#"><img src="images/darktwitter.svg" alt="icon taken from figma"></a></li>
-                    <li><a href="#"><img src="images/darkm.svg" alt="icon taken from figma"></a></li>
+                    <li><a href="#"><img src="images/darkgithub.svg" alt="dark github icon taken from figma"></a></li>
+                    <li><a href="#"><img src="images/darkin.svg" alt="dark in icon taken from figma"></a></li>
+                    <li><a href="#"><img src="images/darkok.svg" alt="dark ok icon taken from figma"></a></li>
+                    <li><a href="#"><img src="images/darktwitter.svg" alt="dark twitter icon taken from figma"></a></li>
+                    <li><a href="#"><img src="images/darkm.svg" alt="dark m icon taken from figma"></a></li>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly change the `alt` attribute value here, it's not important to the user to know where is this image comes from, you can just set it to `github icon` as an example.

File: index.html [hunk 5]
```diff
@@ -36,10 +36,10 @@ <h2 class="profession">I’am a Software Developer</h2>
...
             <ul class="media front">
                 <li><a href="#"><img src="images/github.svg" alt="github icon taken from figma"></a></li>
-                <li><a href="#"><img src="images/in.svg" alt="icon taken from figma"></a></li>
-                <li><a href="#"><img src="images/ok.svg" alt="icon taken from figma"></a></li>
-                <li><a href="#"><img src="images/twitter.svg" alt="icon taken from figma"></a></li>
-                <li><a href="#"><img src="images/m.svg" alt="icon taken from figma"></a></li>
+                <li><a href="#"><img src="images/in.svg" alt="in icon taken from figma"></a></li>
+                <li><a href="#"><img src="images/ok.svg" alt="ok icon taken from figma"></a></li>
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter icon taken from figma"></a></li>
+                <li><a href="#"><img src="images/m.svg" alt="m icon taken from figma"></a></li>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly change the `alt` attribute value here, it's not important to the user to know where is this image comes from, you can just set it to `github icon` as an example. And change the rest of the images in your page

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly enhance your color contrast at the buttons on your page. You can use the [website](https://webaim.org/resources/contrastchecker/) to check your colors and chose the right background color and right color font. And this [website](https://www.aditus.io/button-contrast-checker/) to check your whole page, you put the deploy link and it will check it for you.

The problem here is when the button hovers 
image

And here too is when the button hovers
image




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (zkr024) [reply index.html]: Thank you @keroloussamy, I'll make sure to apply the proposed changes.

**Review by Hamzaoutdoors — APPROVED** [review]
Hi team, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**